Editor/ChannelPackerTool.Editor.asmdef

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This assembly definition just helps isolate the plugin code into its own assembly, which also means it doesn't need to be recompiled when the larger project has code changes.